docs/index.rst

```diff
@@ -27,6 +27,7 @@ those special cases directly into fromager.
    :maxdepth: 2
 
    using.md
+   quickstart.rst
```

Comment:
Please keep quickstart.rst before using.md for users scanning the documentation. A "Quick Start" is typically the first thing new users look for.